packages/react-native-bpk-component-spinner/src/BpkSpinner.js

```diff
@@ -79,9 +89,9 @@ const BpkSpinner = (props: Props) => {
 
   // Validate that spinner is themeable and correct theming attribute has been
   // supplied. If not, disable theming.
-  if (themeAttributes && spinnerType !== 'primary') {
-    themeAttributes = null;
-  }
+  // if (themeAttributes && spinnerType !== 'primary') {
+  //   themeAttributes = null;
+  // }
```

Comment:
Should probably remove this altogether if it's commented out, or if you're leaving it in because we intend it to be added back in the future just add a quick 'TODO uncomment when grey theming no longer needed'.